normalizer/tests/test_api_comparison.py

```
@@ -0,0 +1,218 @@
+        assert 20 <= jageocoder_lat <= 46, f"Jageocoder latitude {jageocoder_lat} is outside Japan range"
+        assert 122 <= jageocoder_lon <= 154, f"Jageocoder longitude {jageocoder_lon} is outside Japan range"
+
+    # 最低1つのAPIで座標が取得できることを検証
+    assert len(valid_apis) >= 1, f"どのAPIでも座標が取得できませんでした。利用可能なAPI: {valid_apis}"
+    print(f"検証成功: {len(valid_apis)}個のAPIで座標を取得 ({', '.join(valid_apis)})")
+
+    # 複数のAPIで座標が取得できた場合は距離を検証
+    if len(valid_apis) >= 2 and gsi_lat is not None and jageocoder_lat is not None:
+        distance = haversine(gsi_lat, gsi_lon, jageocoder_lat, jageocoder_lon)
+        print(f"API間の距離: {distance:.1f}m")
+        # 距離が異常に大きくないことを検証（50km以内）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)